app/controllers/donors_controller.rb

```diff
@@ -1,12 +1,23 @@
 class DonorsController < ApplicationController
+  include ActionView::Helpers::NumberHelper
   # GET donors/:id
   def show
     @donor = find_donor(params[:id])
   end
 
+  # GET /donors
+  def index
+    @donors = Donor.all
+    @total_amt = total_amt_donated
+  end
+
   private
 
   def find_donor(id)
     Donor.find(id)
   end
+
+  def total_amt_donated
```

Comment:
Let's split this logic into two concerns:

1. Sum all the donations.
2. Display it nicely.

We'll do the first one in the model. Thanks to ActiveRecord, there's an easier way to do this:

```
# app/models/donor.rb
def total_donations
  donations.sum(:amount)
end
```

The formatting with `#number_with_delimiter` should be done in the view template for now. 😀